src/_experimental.scss

```diff
@@ -159,3 +159,138 @@
 		}
 	}
 }
+
+
+.provisional {
+	.alert-info > :first-child::before,
+	.alert-success > :first-child::before,
+	.alert-warning > :first-child::before,
+	.alert-danger > :first-child::before {
+		content: none;
+	}
+
+	.alert {
+		background-clip: content-box;
+		background-color: inherit;
+		border-left: 6px solid #000;
+		margin-bottom: 23px;
+		margin-left: 10px;
+		padding: 0 0 0 15px;
+
+		> {
+			p, .ul, .ol {
+				margin-bottom: 0px;
+			}
+
+			* {
+				margin-left: 15px;
+			}
+		}
+
+		details {
+			margin-left: .5em;
+			padding-top: 15px;
+		}
+
+		> {
+			:first-child:not(details) {
+				margin-top: auto;
+				padding-top: 15px;
+			}
+
+			:last-child {
+				padding-bottom: 25px;
+			}
+		}
+
+		&:before {
+			font-family: "Glyphicons Halflings";
+			font-size: 26px;
+			line-height: 2.3em;
+			margin-left: -1.27em;
+			padding: 2px;
+			position: absolute;
+		}
+
+		> :first-child {
+			margin-left: 15px;
+		}
+	}
+
+	.alert-info {
+		border-color: #269abc;
+		border-image: linear-gradient(to bottom, #269abc 16px, #269abc 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, #269abc 48px, #269abc 48px) 1 100%;
+
+
+		&::before {
+			color: #269abc;
+			content: "\e086";
+		}
+
+		&.alert-bg {
+			background-color: #d7faff;
+
+			details > summary {
+				color: #245269;
+				font-weight: 700;
+			}
+		}
+	}
+
+	.alert-success {
+		border-color: #278400;
+		border-image: linear-gradient(to bottom, #278400 16px, #278400 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, #278400 48px, #278400 48px) 1 100%;
+
+		&::before {
+			color: #278400;
+			content: "\e084";
+		}
+
+		&.alert-bg {
+			background-color: #d8eeca;
+
+			details > summary {
+				color: #2b542c;
+				font-weight: 700;
+			}
+		}
+	}
+
+	.alert-warning {
+		border-color: #f90;
+		border-image: linear-gradient(to bottom, #f90 16px, #f90 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, #f90 48px, #f90 48px) 1 100%;
+
+		&::before {
+			color: #f90;
+			content: "\e107";
+		}
+
+		&.alert-bg {
+			background-color: #f9f4d4;
+
+			details > summary {
+				color: #66512c;
+				font-weight: 700;
+			}
+		}
+	}
+
+	.alert-danger {
+		border-color: #d3080c;
+		border-image: linear-gradient(to bottom, #d3080c 16px, #d3080c 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, #d3080c 48px, #d3080c 48px) 1 100%;
+
+		&::before {
+			color: #d3080c;
+			content: "\e101";
+		}
+
+		&.alert-bg {
+			background-color: #f3e9e8;
+
+			details > summary {
+				color: #843534;
+				font-weight: 700;
```

Comment:
It could be preferable to use mixins instead of copying the same rule. 

Here an example:
How to define a mixin https://github.com/wet-boew/wet-boew/blob/master/src/plugins/multimedia/_base.scss#L5 
How to use it: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/multimedia/_base.scss#L104

Reply:
Should I put the mixin variables at the top of the experimental.scss file? Or above the alerts section in the experimental.scss file? Or in the variables.scss/somewhere else in GCWeb?

Reply:
@duboisp I haven't finished this - would be great if you could advise where to put them and I'll update this tomorrow (Thursday)

Reply:
I've gone ahead and defined them at the top of the _experimental.scss sheet. We can move them elsewhere if needed...

Should be ready now.

Reply:
The mixin is only used by SASS during the compilation into CSS. So yes it is correct at the top of the file.